Failure was due to the warning

```
/Users/buildnode/jenkins/workspace/swift-PR-macos/branch-main/swift/test/ClangImporter/objc_async.swift:127:3: error: unexpected warning produced: conformance of 'NonSendableClass' to 'Sendable' is unavailable
  takesSendable(nonSendableClass)     // expected-FIXME-warning{{something about missing conformance}}
  ^
```

CI apparently doesn't pick up branches made to the same repo, so we will have to wait for #40341 to merge before this will pass.

--- a/include/swift/AST/Attr.def
+++ b/include/swift/AST/Attr.def
@@ -700,7 +700,7 @@ CONTEXTUAL_SIMPLE_DECL_ATTR(_const, CompileTimeConst,
   126)
 
 SIMPLE_DECL_ATTR(_unavailableFromAsync, UnavailableFromAsync,
-  OnFunc | OnConstructor |
+  OnFunc | OnConstructor | UserInaccessible |
   ABIStableToAdd | ABIStableToRemove |
   APIBreakingToAdd | APIStableToRemove,
   127)

--- a/lib/Serialization/ModuleFormat.h
+++ b/lib/Serialization/ModuleFormat.h
@@ -56,7 +56,7 @@ const uint16_t SWIFTMODULE_VERSION_MAJOR = 0;
 /// describe what change you made. The content of this comment isn't important;
 /// it just ensures a conflict if two people change the module format.
 /// Don't worry about adhering to the 80-column limit for this line.
-const uint16_t SWIFTMODULE_VERSION_MINOR = 646; // hasHermeticSealAtLink option
+const uint16_t SWIFTMODULE_VERSION_MINOR = 647; // _unavailableFromAsync attr
 
 /// A standard hash seed used for all string hashes in a serialized module.
 ///

--- a/test/IDE/complete_decl_attribute.swift
+++ b/test/IDE/complete_decl_attribute.swift
@@ -72,7 +72,6 @@ struct MyStruct {}
 // KEYWORD2-NEXT:             Keyword/None:                       transpose[#Func Attribute#]; name=transpose
 // KEYWORD2-NEXT:             Keyword/None:                       noDerivative[#Func Attribute#]; name=noDerivative
 // KEYWORD2-NEXT:             Keyword/None:                       Sendable[#Func Attribute#]; name=Sendable
-// KEYWORD2-NEXT:             Keyword/None:                       _unavailableFromAsync[#Func Attribute#]; name=_unavailableFromAsync
 // KEYWORD2-NOT:              Keyword
 // KEYWORD2:                  Decl[Struct]/CurrModule:            MyStruct[#MyStruct#]; name=MyStruct
 // KEYWORD2:                  End completions
@@ -156,7 +155,6 @@ struct _S {
 // ON_INIT-DAG: Keyword/None:                       inlinable[#Constructor Attribute#]; name=inlinable
 // ON_INIT-DAG: Keyword/None:                       usableFromInline[#Constructor Attribute#]; name=usableFromInline
 // ON_INIT-DAG: Keyword/None:                       discardableResult[#Constructor Attribute#]; name=discardableResult
-// ON_INIT-DAG: Keyword/None:                       _unavailableFromAsync[#Constructor Attribute#]; name=_unavailableFromAsync
 // ON_INIT: End completions
 
   @#^ON_PROPERTY^# var foo
@@ -198,7 +196,6 @@ struct _S {
 // ON_METHOD-DAG: Keyword/None:                       transpose[#Func Attribute#]; name=transpose
 // ON_METHOD-DAG: Keyword/None:                       Sendable[#Func Attribute#]; name=Sendable
 // ON_METHOD-DAG: Keyword/None:                       noDerivative[#Func Attribute#]; name=noDerivative
-// ON_METHOD-DAG: Keyword/None:                       _unavailableFromAsync[#Func Attribute#]; name=_unavailableFromAsync
 // ON_METHOD-NOT: Keyword
 // ON_METHOD: Decl[Struct]/CurrModule:            MyStruct[#MyStruct#]; name=MyStruct
 // ON_METHOD: End completions
@@ -260,7 +257,6 @@ struct _S {
 // ON_MEMBER_LAST-DAG: Keyword/None:                       transpose[#Declaration Attribute#]; name=transpose
 // ON_MEMBER_LAST-DAG: Keyword/None:                       noDerivative[#Declaration Attribute#]; name=noDerivative
 // ON_MEMBER_LAST-DAG: Keyword/None:                       Sendable[#Declaration Attribute#]; name=Sendable
-// ON_MEMBER_LAST-DAG: Keyword/None:                       _unavailableFromAsync[#Declaration Attribute#]; name=_unavailableFromAsync
 // ON_MEMBER_LAST-NOT: Keyword
 // ON_MEMBER_LAST: Decl[Struct]/CurrModule:            MyStruct[#MyStruct#]; name=MyStruct
 // ON_MEMBER_LAST-NOT: Decl[PrecedenceGroup]
@@ -311,7 +307,6 @@ func dummy2() {}
 // KEYWORD_LAST-DAG: Keyword/None:                       transpose[#Declaration Attribute#]; name=transpose
 // KEYWORD_LAST-DAG: Keyword/None:                       noDerivative[#Declaration Attribute#]; name=noDerivative
 // KEYWORD_LAST-DAG: Keyword/None:                       Sendable[#Declaration Attribute#]; name=Sendable
-// KEYWORD_LAST-DAG: Keyword/None:                       _unavailableFromAsync[#Declaration Attribute#]; name=_unavailableFromAsync
 // KEYWORD_LAST-NOT: Keyword
 // KEYWORD_LAST: Decl[Struct]/CurrModule:            MyStruct[#MyStruct#]; name=MyStruct
 // KEYWORD_LAST:                  End completions

